nat-lab/tests/mesh_api.py

```diff
@@ -402,3 +406,29 @@ def config_dynamic_nodes(
         assert (len(node_configs) + current_node_list_len) == len(self.nodes)
 
         return tuple(list(self.nodes.values())[current_node_list_len:])
+
+
+def _start_tcpdump_on_vpn(server_name):
+    if os.environ.get("NATLAB_SAVE_LOGS") is not None:
+        # First make sure that no leftover processes/files will not interfere
+        cmd = f"docker exec --privileged {server_name} killall tcpdump"
+        os.system(cmd)
+        cmd = f"docker exec --privileged {server_name} rm {PCAP_FILE_PATH}"
+        os.system(cmd)
+
+        cmd = f"docker exec -d --privileged {server_name} tcpdump -i any -U -w {PCAP_FILE_PATH}"
+        os.system(cmd)
+
+
+def stop_tcpdump_on_vpns():
+    if os.environ.get("NATLAB_SAVE_LOGS") is None:
+        return
```

Comment:
Nit: the can be made more consistent by either having both use an early return (like `stop_tcpdump_on_vpns`) or both do the logic inside the if-block (like `_start_tcpdum_on_vpn`)